src/middleware/auth.js

```diff
@@ -0,0 +1,44 @@
+const jwt = require('jsonwebtoken');
+
+const { ACCESS_TOKEN_SECRET, REFRESH_TOKEN_SECRET } = require('../config/config');
+
+const accessTokenSecret = ACCESS_TOKEN_SECRET;
+const refreshTokenSecret = REFRESH_TOKEN_SECRET;
```

Comment:
I think this was how it was written before, but there's really no reason to redefine `accessTokenSecret` and `refreshTokenSecret`. We can delete these two lines, and then directly use `ACCESS_TOKEN_SECRET` and `REFRESH_TOKEN_SECRET` in the code.